### DISCUSSION TITLE
when is version2.3.0 expected to be released?

### DISCUSSION THREAD

**Discussion by RenJiangZhou2163** [nautobot/nautobot#5925] [Q&A]
Translated to English: I see that the module type feature will be released in version 2.3.0, when is it expected to be released? After this feature goes live, if a module is added to an existing device type, can the device inherit this module?

**Comment by glennmatthews** [comment]
A beta candidate of 2.3 is expected in the next two weeks, with the release to follow soon after. You can try out the current code at https://next.demo.nautobot.com/.

Module bay templates will work like other device type component templates; for example, when creating a *new* device from a device type, it will automatically inherit any module bays defined on the device type, but changes to a device type will not automatically result in any changes to existing devices using that device type.
